src/Tests/Integration/fixtures/banks/visaCal/03-username-tab/frame-12.html

```
@@ -0,0 +1,56 @@
+<!doctype html>
+<html lang="en" style="margin: 0; padding: 0; height: 100%">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=0.1" />
+    <title> </title>
```

HTMLHint (1.9.2)

[error] <title></title> must not be empty.

(title-require)